main.go

```diff
@@ -99,8 +102,21 @@ func realMain(echoChan chan<- *echo.Echo) int {
 		return 5
 	}
 
+	// Plugin Manager
+	homeDir, _ := os.UserHomeDir()
+	pluginExePath, _ := os.Executable()
+	pluginManager := &shared.PluginManager{
+		PluginDirs: []string{
+			filepath.Join(homeDir, ".logproxy/plugins"),
+			pluginExePath,
+		},
+	}
+	if err := pluginManager.Discover(); err == nil {
+		_ = pluginManager.LoadAll()
```

Comment:
Is there a log entry that can be turned on at DEBUG mode to troubleshoot plugin loading?

Reply:
The first few releases will have debugging enabled. Currently you see the full load/handshake/etc process for the plugin subsystem. Logs from plugins are also available, which is pretty neat!